src/reducers/meeting_options_reducer.js

```diff
@@ -4,8 +4,7 @@ const INITIAL_STATE = [];
 export default (state = INITIAL_STATE, action) => {
   switch(action.type){
     case FETCH_MEETING_OPTIONS:
-      // return action.payload.data;
-      return [...state, ...action.payload.data];
+      return action.payload.data;
```

Comment:
Should we do the same `...state` thing here that we're doing elsewhere? Is there some reason not to do so?